tutorials/advanced_features/Local addressability with DMM.ipynb

```diff
@@ -13,8 +13,12 @@
    "metadata": {},
```

Comment:
<p>Seems to be missing a space before "weights" here</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pasqal-io/Pulser/pull/610/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Reply:
<p>Weird, it seems to be a bug in review nb...</p>